<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugDialog</class>
 <widget class="QDialog" name="DebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QPlainTextEdit" name="pt_test"/>
   </item>
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="pt_comm"/>
   </item>
   <item row="0" column="1">
    <widget class="QPlainTextEdit" name="pt_user"/>
   </item>
   <item row="0" column="2">
    <widget class="QPlainTextEdit" name="pt_gui"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pb1">
     <property name="text">
      <string>Wyczyść</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb2">
     <property name="text">
      <string>Wyczyść</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pb3">
     <property name="text">
      <string>Wyczyść</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pb4">
     <property name="text">
      <string>Wyczyść</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
